engine/runtime/src/main/java/org/enso/interpreter/instrument/MethodCallsCache.java

```diff
@@ -20,6 +20,11 @@ public void setExecuted(UUID call) {
     callsExecuted.add(call);
   }
 
+  /** @return the set of executed calls. */
```

Comment:
What is an executed call?

Reply:
Executed call is a function call that we captured in the instrument during the program execution.

Reply:
Is this defined somewhere?

Reply:
There is some info in the class definition

```java
/**
 * A storage for executed methods.
 *
 * <p>Tracks the encountered method calls during the invocation of the stack frame. Method calls
 * cache is not preserved between the program invocations.
 */
public final class MethodCallsCache {
```